deyemiobaa/hello_react_rails_app
# Hello Rails React App

Hello Reviewer 🙋🏽‍♂️

#### In this PR I;
- Set up a react-rails app with `jsbundling-rails`
- Set up a root path to render react component
- Set up an API route to fetch greetings from the database

#### How to test this app;
- update the database.yml file to include a username and password for your local db
- Run `bundle install` and `yarn install` to add project gems and dependencies
- Run `rails db:setup` to generate migrations
- Run `rails db:seed` to add default data
- Run `./bin/dev` to start the server and bundle the javascript files
- Open `localhost:5000` to see live app
